core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/shared/ui/dot-experiments-details-table/dot-experiments-details-table.component.ts

```diff
@@ -34,7 +38,13 @@ export class DotExperimentsDetailsTableComponent {
 
     //** List of data to display, without templates, use the index of the objet as a header */
     @Input()
-    data!: Array<unknown>;
+    data!: DotExperimentSummary[];
+
+    @Input()
+    loading = false;
+
+    @Input()
+    empty = false;
```

Comment:
Does this empty is not just data being undefined? flags as inputs are normally kinda red flag.

Reply:
It's a calc done in the store. `results?.sessions.total === 0` by now is 0, but it will can change. I changed the selector name to `hasEnoughSessions$`